extensions/kubectl-cli/src/download.ts

```diff
@@ -47,6 +47,7 @@ export class KubectlDownload {
   async promptUserForVersion(currentKubectlTag?: string): Promise<KubectlGithubReleaseArtifactMetadata> {
     // Get the latest releases
     let lastReleasesMetadata = await this.kubectlGitHubReleases.grabLatestsReleasesMetadata();
+    lastReleasesMetadata.sort((a, b) => rcompare(a.tag, b.tag));
```

Comment:
avoid using `.sort` please have a preference for `.toSort`

Reply:
why should we not use sort ? here we want to sort the current object, not get a copy

Reply:
IMHO this is to be avoid has it change the object itself,  which in this case is fine, but could have unwanted side effect in other scenarios

Reply:
This is not blocking I do not have a strong opinion

Reply:
I think the wrong pattern is

```
const sorted = myArray.sort()
```

but this is is monitored by https://sonarsource.github.io/rspec/#/rspec/S4043/javascript

Reply:
Oh yes I see, thanks